stale_pr (#13)
ansible/ansibotmini: README.md

| `pre_azp`                | Pull requests            | This PR was last tested before migration to Azure Pipelines.                                                                                                                                                                                                                                                                                             |
| `stale_ci`               | Pull requests            | The CI result is older than a week and needs to be re-run. Close and re-open this PR to get it retested.                                                                                                                                                                                                                                                 |

> | `stale_pr`               | Pull requests            | This PR is based on a commit that is more than one year old.                                                                                                                                                                                                                                                                                             |

Reviewer: I think it would be more accurate to describe `stale_pr` as follows:

```suggestion
| `stale_pr`               | Pull requests            | This PR has not been pushed to for more than one year.                                                                                                                                                                                                                                                                                             |
```
